pkg/util/proto/openapi_suite_test.go

```diff
@@ -35,15 +35,15 @@ func TestOpenapi(t *testing.T) {
 // https://github.com/jstemmer/go-junit-report/issues/31
 type newlineReporter struct{}
 
-func (newlineReporter) SpecSuiteWillBegin(config GinkgoConfigType, summary *SuiteSummary) {}
+func (newlineReporter) SuiteWillBegin(config GinkgoConfigType, summary *types.SuiteSummary) {}
 
-func (newlineReporter) BeforeSuiteDidRun(setupSummary *SetupSummary) {}
+func (newlineReporter) BeforeSuiteDidRun(setupSummary *types.SetupSummary) {}
 
-func (newlineReporter) AfterSuiteDidRun(setupSummary *SetupSummary) {}
+func (newlineReporter) AfterSuiteDidRun(setupSummary *types.SetupSummary) {}
 
-func (newlineReporter) SpecWillRun(specSummary *SpecSummary) {}
+func (newlineReporter) SpecWillRun(specSummary *types.SpecSummary) {}
 
-func (newlineReporter) SpecDidComplete(specSummary *SpecSummary) {}
+func (newlineReporter) SpecDidComplete(specSummary *types.SpecSummary) {}
 
-// SpecSuiteDidEnd Prints a newline between "35 Passed | 0 Failed | 0 Pending | 0 Skipped" and "--- PASS:"
-func (newlineReporter) SpecSuiteDidEnd(summary *SuiteSummary) { fmt.Printf("\n") }
+// SuiteDidEnd Prints a newline between "35 Passed | 0 Failed | 0 Pending | 0 Skipped" and "--- PASS:"
+func (newlineReporter) SuiteDidEnd(summary *types.SuiteSummary) { fmt.Printf("\n") }
```

Comment:
you can just remove this newline reporter If you want,  pls see: https://github.com/kubernetes/kubernetes/pull/110319

Reply:
Thanks! I will remove it